core/domain/exp_jobs_one_off_test.py

```diff
@@ -778,108 +1483,264 @@ class ExplorationContentValidationJobForTextAngularTests(
+            'content': {},
+            'default_outcome': {},
+            'hint1': {},
         }
-        state1.update_content(content1_dict)
-        state2.update_content(content2_dict)
+
+        state1.update_interaction_hints(hint_list1)
+        state1.update_content_ids_to_audio_translations(
+            content_ids_to_audio_translations_dict1)
+
+        state2.update_interaction_hints(hint_list2)
+        state2.update_content_ids_to_audio_translations(
+            content_ids_to_audio_translations_dict2)
+
         exp_services.save_new_exploration(self.albert_id, exploration)
 
-        # Start validation job on sample exploration.
-        job_id = exp_jobs_one_off.ExplorationContentValidationJobForTextAngular.create_new() # pylint: disable=line-too-long
+        # Start HintsAuditOneOff job on sample exploration.
+        job_id = exp_jobs_one_off.HintsAuditOneOffJob.create_new()
+        exp_jobs_one_off.HintsAuditOneOffJob.enqueue(job_id)
+        self.process_and_flush_pending_tasks()
+
+        actual_output = exp_jobs_one_off.HintsAuditOneOffJob.get_output(job_id)
+        expected_output = [
+            '[u\'1\', [u\'exp_id0 State2\']]',
+            '[u\'2\', [u\'exp_id0 State1\']]'
+        ]
+        self.assertEqual(actual_output, expected_output)
+
+    def test_no_action_is_performed_for_deleted_exploration(self):
+        """Test that no action is performed on deleted explorations."""
+
+        exploration = exp_domain.Exploration.create_default_exploration(
+            self.VALID_EXP_ID, title='title', category='category')
+
+        exploration.add_states(['State1'])
+
+        state1 = exploration.states['State1']
+
+        hint_list = [{
+            'hint_content': {
+                'content_id': 'hint1',
+                'html': '<p>Hello, this is html1 for state1</p>'
+            }
+        }, {
+            'hint_content': {
+                'content_id': 'hint2',
+                'html': '<p>Hello, this is html2 for state1</p>'
+            }
+        }]
+
+        content_ids_to_audio_translations_dict = {
+            'content': {},
+            'default_outcome': {},
+            'hint1': {},
+            'hint2': {}
+        }
+
+        state1.update_interaction_hints(hint_list)
+        state1.update_content_ids_to_audio_translations(
+            content_ids_to_audio_translations_dict)
+
+        exp_services.save_new_exploration(self.albert_id, exploration)
+
+        exp_services.delete_exploration(self.albert_id, self.VALID_EXP_ID)
+
+        # Start HintsAuditOneOff job on sample exploration.
+        job_id = exp_jobs_one_off.HintsAuditOneOffJob.create_new()
+        exp_jobs_one_off.HintsAuditOneOffJob.enqueue(job_id)
+        self.process_and_flush_pending_tasks()
+
+        self.assertEqual(
+            exp_jobs_one_off.HintsAuditOneOffJob.get_output(job_id),
+            []
+        )
+
+
+class TextAngularValidationAndMigrationTests(test_utils.GenericTestBase):
```

Comment:
@bansalnitish is this job useful anymore? If not should we remove it? 

(Asking because we migrated successfully from Text Angular to CKEditor last GSoC).

Reply:
Yes, right. This is not useful anymore. I'll make a PR and remove this job (and tests too if this gets in :) ) if you say.